main.go

```diff
@@ -73,41 +52,53 @@ func main() {
 	b.Watch(wd)
 }
 
-func parseConfig() (config, error) {
-	var c config
-
-	// if we have any cliCmds, set them to our build phase
-	c.Build = cliCmds
-
-	// if build phase is still empty try and find the snag.yml file
-	if len(c.Build) == 0 {
-		in, err := ioutil.ReadFile(".snag.yml")
-		if err != nil {
-			return c, errors.New("Could not find '.snag.yml' in your current directory")
-		}
-
-		if err := yaml.Unmarshal(in, &c); err != nil {
-			return c, fmt.Errorf("Could not parse yml file. %s\n", err)
-		}
+func handleSubCommand(cmd string) error {
+	switch flag.Arg(0) {
+	case "init":
+		return initSnag()
+	default:
+		flag.Usage()
+		return nil
 	}
+}
 
-	// if both script and build are specified
-	// blow up and tell the user to use build
-	if len(c.Script) != 0 && len(c.Build) != 0 {
-		return c, errors.New("Cannot use 'script' and 'build' together. The 'script' tag is deprecated, please use 'build' instead.")
+func initSnag() error {
+	if _, err := os.Stat(SnagFile); err == nil {
+		return errors.New("snag file already exists")
 	}
 
-	// if script has something, tell the user it's deprecated
-	// and set whatever its contents are to build
-	if len(c.Script) != 0 {
-		c.DepWarnning += "*\tThe use of 'script' in the yaml file has been deprecated and will be removed in the future.\n\tPlease start using 'build' instead.\n\n"
-		c.Build = c.Script
+	f, err := os.Create(SnagFile)
+	if err != nil {
+		return err
 	}
-
-	if len(c.Build) == 0 {
-		return c, errors.New("You must specify at least 1 command.")
+	defer f.Close()
+
+	tmpl := `---
```

Comment:
I thought about that, the reason I went ahead and defined it only for `init` was because the command will only be ran once and be done. If we define it as a `const` it'll live the entire life of the program even when we run `snag` and it's not being used. After that, do you still think it makes sense for it being a `const`. If you do I'll make the change.